### PR TITLE
Adds docs on how to enable Live Reload in a starter app

### PR DESCRIPTION
**Pull request by coltborg**
[#183429068](https://www.pivotaltracker.com/story/show/183429068)

This feature is really a configuration change in IntelliJ. Code wise, I documented the steps and then removed some application configuration that isn't needed as it wasn't contributing to allowing Live Reload to work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,19 +145,37 @@ After cloning the repository, run `scripts/setup.sh` from the root of the repo's
...
 
+#### Setup Live Reload
+
+Live Reload is very helpful when making many changes to HTML templates, CSS, or JavaScript. Here are instructions on how to get IntelliJ to reload resources and have the LiveReload browser extension automatically reload the browser tab for you.
+
+* Download live reload extension in your browser of choice:
+    * [Firefox extension](https://addons.mozilla.org/en-US/firefox/addon/livereload-web-extension/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search)
+    * [Chrome extension](https://chromewebstore.google.com/detail/livereload/jnihajbhpnppcggbcgedagnkighmdlei)
+* Restart your browser after install
+* Go to `http://localhost:8080/`
```
Comment (enyia21) [inline README.md:169]: Should we alert developers that they need to start up their applications before visiting `http://localhost:8080`?

**Review by enyia21 — CHANGES_REQUESTED** [review]
There is only one change that I'd like to see made

File: README.md (hunk 1 above)
Comment (coltborg) [reply README.md]: I moved around the structure so that it makes more sense, to your point. Thank you!

**Review by enyia21 — APPROVED** [review]
Great

**Pull request merged**